src/test/metersphere/templates/03-modules/mysql.yaml

```diff
@@ -0,0 +1,77 @@
+{{ if .Values.mysql.enabled }}
```

Comment:
is it possible to use mysql chart as a dependency?

Reply:
It's better not to modify this. The original repo used this way, refer to https://github.com/metersphere/helm-chart/tree/master/helm-chart-sources/metersphere. Changing it maybe bring more tests working.